rules/falco_rules.yaml

```diff
@@ -2872,11 +2872,19 @@
 - list: k8s_client_binaries
   items: [docker, kubectl, crictl]
 
+- macro: user_known_k8s_ns_kube_system_images
```

Comment:
Sorry, I meant to use a a `list` instead of a `macro`

```
- list: user_known_k8s_ns_kube_system_images
  items: [k8s.gcr.io/fluentd-gcp-scaler, k8s.gcr.io/node-problem-detector/node-problem-detector]

- macro: user_known_k8s_client_container
  condition: (k8s.ns.name="kube-system" and container.image.repository in (user_known_k8s_ns_kube_system_images)) or container.image.repository=mcr.microsoft.com/aks/hcp/hcp-tunnel-front
```

Reply:
I understood list was an alternative to the condition.

Thanks